### ISSUE TITLE
META LIST: potentially offensive terms

### ISSUE THREAD

**Issue by colleenXu** [NCATSTranslator/Feedback#286]
put these here!

**Comment by jeffhhk** [comment]
On this world-writable spreadsheet: https://docs.google.com/spreadsheets/d/1UR2eplHBvFRwaSIVOhlB44wpfNPY1z7AVzUkqzDqIWA/edit#gid=0

**Comment by remontoire-pac** [comment]
![image](https://github.com/NCATSTranslator/Feedback/assets/2120526/3d8812b6-082d-49a2-bf47-a2f2b14b9df7)

ONE OF THE RESULTS

![image](https://github.com/NCATSTranslator/Feedback/assets/2120526/4530fce5-3738-4a79-8e03-da306851f2aa)

**Comment by remontoire-pac** [comment]
![image](https://github.com/NCATSTranslator/Feedback/assets/2120526/0eca8075-1f86-4ff7-9578-ac571bc45622)

**Comment by remontoire-pac** [comment]
![image](https://github.com/NCATSTranslator/Feedback/assets/2120526/68ab2133-d3b6-4237-a09f-f74a1d54c87a)

**Comment by karafecho** [comment]
Late pregnancy (as a disease)
PK = f9288cda-fdb6-422e-bd41-a256c9d700db

_*https://arax.ncats.io/ listed as the only source._

![image](https://github.com/NCATSTranslator/Feedback/assets/26254388/7ee07dff-3f9f-4277-a34a-18e0c1b85e50)

**Comment by Genomewide** [comment]
Found additional potentially sensitive terms by matching in mondo and then looking for neighbors

![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/63f35d90-7df4-4708-b36d-8d3e44aae5f8)

**Comment by gaurav** [comment]
All subclasses of "sexual and gender identity disorders" are now on the Blocklist and are blocked on https://ui.ci.transltr.io/. Neither "suicide" nor "gender" returns any autocomplete results on https://ui.ci.transltr.io/. I'll go through the Google Sheet mentioned in https://github.com/NCATSTranslator/Feedback/issues/286#issuecomment-1579007519 to see if anything else got left out, but I think most of the autocomplete blocks required by this ticket have happened.

@MarkDWilliams Is there a good ticket to track blocking results as suggested by some of the comments on this issue?

**Comment by gaurav** [comment]
I've moved all the Autocomplete Blocklists terms to the [Blocklist Spreadsheet](https://docs.google.com/spreadsheets/d/1UR2eplHBvFRwaSIVOhlB44wpfNPY1z7AVzUkqzDqIWA/edit#gid=0) -- and we have [another ticket](https://github.com/NCATSTranslator/Blocklists/issues/33) for checking if any of them are still around -- and the few Result Blocklists terms have been reported at https://github.com/NCATSTranslator/Blocklists/issues/44. I'm going to go ahead and close this ticket, but please reopen if I left out anything.
